pkg/spartan/config.go

```diff
@@ -40,6 +40,10 @@ var IPs = []net.IPNet{
 		IP:   net.IPv4(198, 51, 100, 3),
 		Mask: net.IPv4Mask(0xff, 0xff, 0xff, 0xff),
 	},
+    net.IPNet{
```

Comment:
Could you run "go fmt" on the file. The structure seems to be mis-aligned.

Reply:
Could you also add a comment as to the need for setting up the IP address for the `kube-dns-proxy`?